sentry-android-fragment/api/sentry-android-fragment.api

```diff
@@ -20,15 +21,33 @@ public final class io/sentry/android/fragment/FragmentLifecycleIntegration : and
 	public fun register (Lio/sentry/IHub;Lio/sentry/SentryOptions;)V
 }
 
+public final class io/sentry/android/fragment/FragmentLifecycleState : java/lang/Enum {
+	public static final field ATTACHED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field CREATED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field DESTROYED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field DETACHED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field PAUSED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field RESUMED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field SAVE_INSTANCE_STATE Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field STARTED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field STOPPED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field VIEW_CREATED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static final field VIEW_DESTROYED Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static fun valueOf (Ljava/lang/String;)Lio/sentry/android/fragment/FragmentLifecycleState;
+	public static fun values ()[Lio/sentry/android/fragment/FragmentLifecycleState;
+}
+
 public final class io/sentry/android/fragment/SentryFragmentLifecycleCallbacks : androidx/fragment/app/FragmentManager$FragmentLifecycleCallbacks {
 	public static final field Companion Lio/sentry/android/fragment/SentryFragmentLifecycleCallbacks$Companion;
 	public static final field FRAGMENT_LOAD_OP Ljava/lang/String;
+	public fun <init> (Lio/sentry/IHub;Ljava/util/Set;Z)V
+	public synthetic fun <init> (Lio/sentry/IHub;Ljava/util/Set;ZILkotlin/jvm/internal/DefaultConstructorMarker;)V
 	public fun <init> (Lio/sentry/IHub;ZZ)V
-	public synthetic fun <init> (Lio/sentry/IHub;ZZILkotlin/jvm/internal/DefaultConstructorMarker;)V
```

Comment:
This is the only "breaking" change I must leave to keep the code looking sane. Instead of `(IHub, Boolean,Boolean)` the default constructor is now `(IHub,Set,Boolean)`. Of course the old one is also present, but as a secondary constructor. Is this ok for you?

Reply:
yeah, as long as we preserve the old ctor, it's fine 👍